corehq/motech/repeaters/models.py

```diff
@@ -773,6 +775,27 @@ def cancel(self):
         self.next_check = None
         self.cancelled = True
 
+    def attempt_forward_now(self):
+        from corehq.motech.repeaters.tasks import process_repeat_record
+
+        if self.next_check > datetime.utcnow():
```

Comment:
Should this check these other things first, in case the record is in an unexpected state?
```py
if not (self.succeeded or self.cancelled or self.next_check is None):
    return
```

I think all of that is redundant with the state of newly created repeat records and the `repeaters/repeat_records_by_next_check` view definition, but those are sufficiently far away from this code that it may make sense to double-check here. What do you think?

Reply:
I think it makes sense, even if only to help someone new to the code understand how this works. (Are you sure about that `not`?)

Reply:
> Are you sure about that `not`?

nope, accidentally reversed the logic along the way somehow.